experimenter/experimenter/experiments/models.py

```diff
@@ -836,7 +837,61 @@ def clone(self, name, user, rollout_branch_slug=None):
         return cloned
 
     def get_changelogs(self):
-        return self.changes.all()
+        changes_by_date = defaultdict(list)
+        date_option = "%I:%M:%S %p"
+        changelogs = list(
+            self.changes.order_by("-changed_on").prefetch_related("changed_by")
+        )
+        len(changelogs) - 1
```

Comment:
we are not storing it in a variable or something, if you don't want to use it we can remove this line